### ISSUE TITLE
Preview not skipping new segment

### ISSUE THREAD

**Issue by PhoenixIV** [ajayyy/SponsorBlock#1381]
I just tried to upload my very first segment to contribute and wanted to use "preview" to see how it looks when it is skipped. However, my segment was not skipped.

I disabled skipping beforehand because it hindered me in creating my own, refined segment. This might be the cause. However I would expect the preview to ignore my no skipping general setting.

![Unbenannt](https://user-images.githubusercontent.com/2091755/176545671-5bfb03da-b859-4d43-b48e-3b1bb9f86c52.JPG)

I found https://github.com/ajayyy/SponsorBlock/issues/915 , which was resolved, so I guess this is suppoesed to sk

**Comment by ajayyy** [comment]
For now, you can hide the individual segments that are messing with your preview using the eye button in the popup

![image](https://user-images.githubusercontent.com/12688112/176560414-3aa2e853-b5d2-4208-91db-6656819fa215.png)

**Comment by PhoenixIV** [comment]
I do not think it would be such big of a deal if all my local, unsubmitted segments are skipped in the preview. What makes you think it would not be okay to skip all unsubmitted segments?
